api/URLUtilsReadOnly.json

```diff
@@ -0,0 +1,718 @@
+{
+  "api": {
+    "URLUtilsReadOnly": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/URLUtilsReadOnly",
+        "support": {
+          "chrome": {
+            "version_added": false,
+            "notes": "Though not grouped in a single abstract interface, these methods are directly available on the interfaces that implement it, if this interface is supported."
```

Comment:
I'm a bit concerned by these abstract interface notes. It seems to me that if the interface is unsupported (i.e., the value is `false`) then nothing meaningful is said about the availability of the methods. I'm inclined to not have any of these notes (especially since the browsers with a non-`false` value don't have this note anyway)